### ISSUE TITLE
SQS alert delivery panic

### ISSUE THREAD

**Issue by austinbyers** [panther-labs/panther#1529]
### Describe the bug

After deploying v1.8.0-beta, we got the following panic when trying to deliver an SQS alert in the snowflake demo acct

```
RequestId: 8b85322a-4730-5058-910f-65ffaf9fdf41 Process exited before completing request
2020/09/11 21:04:55 unexpected EOF
2020/09/11 21:04:55 unexpected EOF
/Users/nhakmiller/go/src/github.com/panther-labs/panther-enterprise/internal/core/alert_delivery/api/send.go:45 +0x111
created by github.com/panther-labs/panther/internal/core/alert_delivery/api.sendAlerts
/usr/local/Cellar/go/1.14/libexec/src/runtime/asm_amd64.s:1373 +0x1 fp=0xc0005d5fb8 sp=0xc0005d5fb0 pc=0x4625e1
runtime.goexit()
/Users/nhakmiller/go/src/github.com/panther-labs/panther-enterprise/internal/core/alert_delivery/api/send.go:103 +0x60f fp=0xc0005d5fb0 sp=0xc0005d5bf8 pc=0xc4033f
github.com/panther-labs/panther/internal/core/alert_delivery/api.sendAlert(0xc0005aa240, 0xc00006eb40, 0x795a739, 0xed6eddd77, 0x0, 0xc00003c0c0)
/Users/nhakmiller/go/src/github.com/panther-labs/panther-enterprise/internal/core/alert_delivery/outputs/sqs.go:46 +0x435 fp=0xc0005d5bf8 sp=0xc0005d58a0 pc=0xbb79d5
github.com/panther-labs/panther/internal/core/alert_delivery/outputs.(*OutputClient).Sqs(0xc000606f60, 0xc0005aa240, 0xc00075b1b0, 0x5958704144495151)
/Users/nhakmiller/go/src/github.com/panther-labs/panther-enterprise/internal/core/alert_delivery/outputs/sqs.go:86 +0x1ec fp=0xc0005d58a0 sp=0xc0005d5820 pc=0xbb7fec
github.com/panther-labs/panther/internal/core/alert_delivery/outputs.(*OutputClient).getSqsClient(0xc000606f60, 0xc000100140, 0x41, 0xc00071e000, 0x2e1)
/usr/local/Cellar/go/1.14/libexec/src/runtime/map_faststr.go:291 +0x3de fp=0xc0005d5820 sp=0xc0005d57b8 pc=0x412f3e
runtime.mapassign_faststr(0xd1f7e0, 0xc000606f00, 0xc00010014c, 0x9, 0x1)
/usr/local/Cellar/go/1.14/libexec/src/runtime/panic.go:1112 +0x72 fp=0xc0005d57b8 sp=0xc0005d5788 pc=0x4332f2
runtime.throw(0xe3965d, 0x15)
goroutine 13 [running]:
internal/poll.(*pollDesc).waitRead(...)
/usr/local/Cellar/go/1.14/libexec/src/internal/poll/fd_poll_runtime.go:87 +0x45
```

**Comment by austinbyers** [comment]
Found the problem: we're using a map of clients by region which is cached globally, but we deliver alerts in parallel. There is a `concurrent map writes` error that happens above the posted stack trace. Thanks @s0l0ist for finding the top-level error, I'll put in the fix

**Comment by austinbyers** [comment]
Fixed by #1530
